redash/worker.py

```diff
@@ -12,9 +12,22 @@
 from redash import create_app, settings
 from redash.metrics import celery as celery_metrics  # noqa
 
+import ssl
+
+redis_ssl_config = None
+if settings.CELERY_BROKER.startswith('rediss'):
+    # TODO make these values configurable
```

Comment:
Kindly do :)

Reply:
Happy to! Do you have any suggestions on how? Should each of these properties be wired up to new settings? What should the defaults be?

Reply:
We configure pretty much everything in `redash/settings/__init__.py`. Since these configuration bits are tightly coupled, it might be a good idea to move the entire config dict over there, and hydrate it from `os.environ`.

Regarding defaults - I'm really only guessing here, but perhaps a good idea is to default to optional cert and include guidance (documentation or links) on how to setup proper certificates for the connection? (/cc @arikfr?)